examples/docs/src/templates/docs.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import { graphql } from "gatsby";
+import MDXRenderer from "gatsby-mdx/mdx-renderer";
+import styled, { injectGlobal } from "react-emotion";
+import { Edit3 } from "react-feather";
+import { Layout, Link } from "$components";
+
+injectGlobal`
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  html, body {
+    font-family: -apple-system,
+      BlinkMacSystemFont,
+      "Segoe UI",
+      "Roboto",
+      "Roboto Light",
+      "Oxygen",
+      "Ubuntu",
+      "Cantarell",
+      "Fira Sans",
+      "Droid Sans",
+      "Helvetica Neue",
+      sans-serif,
+      "Apple Color Emoji",
+      "Segoe UI Emoji",
+      "Segoe UI Symbol";
+
+    font-size: 16px;
+  }
+
+  a {
```

Comment:
global link style seems weird (especially on a tag), should move it into the link component or something like that.

Reply:
this can be done as future work.